chrome/content/zotero/xpcom/zotero.js

```diff
@@ -381,7 +381,26 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 					return false;
 				}
 			}
-			
+			if(Zotero.File.isDropboxDirectory(dataDir.path) && Zotero.Prefs.get('warnDropboxDataDir')){
+				let check = {value: false};
+				ps = Services.prompt;
+				index = ps.confirmEx(null,
```

Comment:
@KarlHegbloom This is a line note on a pull request. If you have a comment on the proposed implementation you can make it on the [ticket](https://github.com/zotero/zotero/issues/437). Don't know what you mean about SMTP, though. In any case, I'm deleting this.